applicationset/generators/git.go

```diff
@@ -194,6 +194,8 @@ func (g *GitGenerator) generateParamsFromGitFile(filePath string, fileContent []
 				}
 			}
 		}
+		params["repoURL"] = gitGenerator.RepoURL
+		params["revision"] = gitGenerator.Revision
```

Comment:
This worries me a little bit. If someone is already using these two keys in their git config files, we'll clobber them.

Two possible suggestions:

1. nest the new params in `"generator"` - it's unlikely anyone is using that key, so we'll avoid clobbering them
2. only replace if not already present - this is problematic if the author wants to enforce that the user can't override these params